public/javascripts/controllers/UserCtrl.js

```diff
@@ -14,5 +14,10 @@ function($scope, users, auth){
   };
 
   loadGoogleCalendarEvents();
+  $scope.goToThisDetails=function(alignToTop){
```

Comment:
1. Is this method same as the one used in angularApp.js
Which one is used ?
Remove the one which is not used.
2. Add a comment above the method 
   - explain what it does
   - what are the parameters passed to it & what should be there expected value